rogue-thi-app/pages/calendar.js

```diff
@@ -56,13 +56,11 @@ export default function Calendar () {
 
   useEffect(() => {
     async function load () {
-      const [campusLifeEvents, thiEvents] = await Promise.all([
-        NeulandAPI.getCampusLifeEvents(),
-        NeulandAPI.getThiEvents()
+      const [campusLifeEvents] = await Promise.all([
```

Comment:
You could simplify this to `const campusLifeEvents = await NeulandAPI.getCampusLifeEvents()`